pom.xml

```diff
@@ -110,6 +118,12 @@
             <version>3.0.2</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.Anon8281</groupId>
+            <artifactId>UniversalScheduler</artifactId>
```

Comment:
This was last updated 2y ago. I would strongly advise against including it. The project itself is MIT licensed.
So you could vendorize it (like I did with the Webhook library)?
+ give credits as is required by MIT.